src/library/Uno.Themes.WinUI.Markup/Themes/FloatingActionButton.cs

```diff
@@ -0,0 +1,237 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryStateOverlayBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabSecondaryStateOverlayBackgroundDisabled", true);
+					}
+				}
+
+				public static class Tertiary
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryForeground")]
+						public static ResourceValue<Brush> Default => new("FabTertiaryForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryForegroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabTertiaryForegroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabTertiaryForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabTertiaryForegroundDisabled", true);
+					}
+
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryBackground")]
+						public static ResourceValue<Brush> Default => new("FabTertiaryBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryBackgroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabTertiaryBackgroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabTertiaryBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabTertiaryBackgroundDisabled", true);
+					}
+				}
+
+				public static class TertiaryStateOverlay
+				{
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryStateOverlayBackground")]
+						public static ResourceValue<Brush> Default => new("FabTertiaryStateOverlayBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryStateOverlayBackgroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabTertiaryStateOverlayBackgroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryStateOverlayBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabTertiaryStateOverlayBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryStateOverlayBackgroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FabTertiaryStateOverlayBackgroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabTertiaryStateOverlayBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabTertiaryStateOverlayBackgroundDisabled", true);
+					}
+				}
+			}
+
+			public static class Styles
```

Comment:
Missing some styles here @lesterbotello to follow the XAML
- MaterialSurfaceFabStyle
- MaterialSecondaryFabStyle
- MaterialTertiaryFabStyle
- MaterialSmallFabStyle
- MaterialSurfaceSmallFabStyle
- MaterialSecondarySmallFabStyle
- MaterialTertiarySmallFabStyle
- MaterialLargeFabStyle
- MaterialSurfaceLargeFabStyle
- MaterialSecondaryLargeFabStyle
- MaterialTertiaryLargeFabStyle